sympy/stats/crv_types.py

```diff
@@ -2320,6 +2320,32 @@ def pdf(self, x):
     def sample(self):
         return random.normalvariate(self.mean, self.std)
 
+    def _sample_numpy(self, **kwargs):
+        mean, std = self.mean, self.std
+        if isinstance(mean, Symbol) or isinstance(std, Symbol):
+            raise ValueError('Cannot sample from Symbolic parameters.')
+
+        numpy = import_module('numpy')
+        if numpy:
+            mean, std = float(mean), float(std)
```

Comment:
Casting may invite errors in computation. Is there any other method for going from `SymPy` objects to `float` objects.

Reply:
Like, using `args` and all that stuff. Please try to remove casting, if possible.

Reply:
I don't have much idea in this area. Maybe @sidhantnagpal clarify this.

Reply:
May be there isn't a way. I tried it out. Comment if you can find out any, otherwise, it's fine to do casting. I think, `SymPy` has its own method like, `__float__` which can avoid errors. Can you check?

Reply:
`__float__` method internally uses `float` to convert to float.
```python
def __float__(self):
    # Don't bother testing if it's a number; if it's not this is going
    # to fail, and if it is we still need to check that it evalf'ed to
    # a number.
    result = self.evalf()
    if result.is_Number:
        return float(result)
    if result.is_number and result.as_real_imag()[1]:
        raise TypeError("can't convert complex to float")
    raise TypeError("can't convert expression to float")
```
So, I think above is fine.

Reply:
Resolve this conversation.